src/app/public/styles/_switch.scss

```diff
@@ -3,7 +3,6 @@ $sky-switch-size: 22px !default;
 .sky-switch {
   cursor: pointer;
   display: inline-flex;
-  white-space: nowrap;
```

Comment:
Wouldn't this cause any checkbox labels that aren't in a `sky-list-view-checklist` to now wrap?

Reply:
Correct. Which is what they are supposed to be doing https://github.com/blackbaud/skyux-forms/issues/19

Reply:
👍